### PR TITLE
Added load.py to readme and requirements

### LINKED ISSUE #63: Update README for load.py
**Description:**
Enhance the README file to provide clear instructions for using the load.py script, including setup, dependencies, and usage examples.

**Outcome:**
An updated README file with comprehensive documentation for the load.py script, making it easier for developers to understand and use the script effectively.

**User Story:**
As a developer, I need clear documentation for the load.py script, so I can quickly understand its purpose, setup requirements, and how to use it without additional guidance.

**Design:**

-     Include a brief description of the script's functionality.
-     Provide prerequisites and installation steps for dependencies.
-     Add step-by-step instructions for running the script, including command-line arguments if applicable.

### PR DESCRIPTION
**Pull request by qrafiq1**
READEME has been updated to include load.py and now a requirements sub-heading. 

Closes #63

### CODE REVIEW

File: pipeline/README.MD [hunk 1]
```diff
@@ -2,11 +2,36 @@
...
-- `test_transform.py` - this test file verifies the main functions in transform.py through unit tests, achieving 82% test coverage with pytest, and ensuring that data transformation is performed accurately without introducing errors.
+- `test_transform.py` - this test file verifies the main functions in `transform.py` through unit tests, achieving 82% test coverage with pytest, and ensuring that data transformation is performed accurately without introducing errors.
+- `load.py` - this file loads takes clean data from transform and loads it into the Microsoft SQL Server hosted on RDS AWS.
+- `test_load.py` - this test file verifies the core functions in `load.py` through unit tests.
 - `schema.sql` - this SQL script establishes a relational database structure within a specified schema to store and manage plant-related information. Known data is seeded to the tables.
 - `reset.sh` - this bash script loads environment variables and utilises them in the running of `schema.sql` in order to create a Microsoft SQL Server database.
 - `connect.sh` - this bash script loads environment variables to connect to the created Microsoft SQL Server database.
```
Comment (keoghrmj) [inline pipeline/README.MD:37]: It's good but what about the `etl.py` script?

**Review by keoghrmj — CHANGES_REQUESTED** [review]
Please can you make sure to explain the `etl.py` script.

File: pipeline/README.MD [hunk 2]
```diff
@@ -2,11 +2,36 @@
...
 - `test_extract.py` - this test file employs patching techniques to mock external dependencies and validate the functionality of `extract.py`, including the correct extraction of plant metrics, while preventing any real-world API calls. 
 - `transform.py` - this file performs data cleaning tasks, such as removing null values, converting columns to appropriate data types, and ensuring numerical consistency by rounding values to predefined precision levels.
-- `test_transform.py` - this test file verifies the main functions in transform.py through unit tests, achieving 82% test coverage with pytest, and ensuring that data transformation is performed accurately without introducing errors.
+- `test_transform.py` - this test file verifies the main functions in `transform.py` through unit tests, achieving 82% test coverage with pytest, and ensuring that data transformation is performed accurately without introducing errors.
+- `load.py` - this file loads takes clean data from transform and loads it into the Microsoft SQL Server hosted on RDS AWS.
+- `test_load.py` - this test file verifies the core functions in `load.py` through unit tests.
```
Comment (FarihaChoudhury) [inline pipeline/README.MD:34]: do you think we should order the files, like keep all the test files at the bottom?

Comment (SurinaCS) [inline pipeline/README.MD:34]: could we find out what percentage the test_load coverage is and add it here?

**Review by SurinaCS — CHANGES_REQUESTED** [review]
suggestion of an edit for consistency.

**Review by SurinaCS — APPROVED** [review]

**Review by keoghrmj — APPROVED** [review]

**Pull request merged**